### PR TITLE
Capability Sets Documentation

### PR DESCRIPTION
**Pull request by wesbland**
Adds README and environment variables to learn about capability sets.

### CODE REVIEW

File: src/mpid/ch4/netmod/ofi/ofi_init.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ categories :
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_LOCAL
       description : >-
+        Capability Sets:
```
Comment (raffenet) [inline src/mpid/ch4/netmod/ofi/ofi_init.h]: What's the intention for adding this tag? IMO it doesn't improve the `README.envvar` content, but I might be missing a use case.

**Comment by wesbland** [comment]
Your changes look fine to me. If you want me to remove the "Capability Sets:" tag, I can take it out. Otherwise this is probably ready to go.

**Comment by raffenet** [comment]
Go ahead and remove it. Good to merge after that.

**Comment by wesbland** [comment]
Done

**Pull request merged**